components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/handlers/grant/RefreshGrantHandler.java

```diff
@@ -240,7 +242,7 @@ private void setPropertiesForTokenGeneration(OAuthTokenReqMessageContext tokReqM
         tokReqMsgCtx.setScope(validationBean.getScope());
         tokReqMsgCtx.getOauth2AccessTokenReqDTO().setAccessTokenExtendedAttributes(
                 validationBean.getAccessTokenExtendedAttributes());
-        if (StringUtils.isNotBlank(validationBean.getTokenBindingReference()) && !NONE
```

Comment:
Is this intentional. If not let's keep as same as all the places. Having the Class will helpful to understand which logic getting executed. Other wise it might difficult when  reading the code.
If there any special reason for this, please share.

Reply:
This was formatted automatically. Will revert this change.